browser/ui/views/tabs/brave_tab.cc

```diff
@@ -88,6 +147,40 @@ void BraveTab::Layout() {
       close_button_->SetButtonPadding({});
     }
   }
+
+  if (!base::FeatureList::IsEnabled(tabs::features::kBraveVerticalTabs)) {
+    return;
```

Comment:
what is it for? I don't see any code below. Shouldn't this check be placed above?

Reply:
Oops, you're right. I don't understand how come this was put here 😭 IsAtMinWidthForVerticalTabStrip() is checking the feature flag, so we don't need any change in this method.